salt/utils/dictupdate.py

```diff
@@ -17,7 +17,7 @@
 log = logging.getLogger(__name__)
 
 
-def update(dest, upd, recursive_update=True, merge_lists=False):
+def update(dest, upd, recursive_update=True, merge_lists=True):
```

Comment:
The modification of these default values concerns me a little bit because I'm worried that it might regress other functions that use this one. Is this necessary?